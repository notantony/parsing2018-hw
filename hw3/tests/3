max A B = if > A B print A print B