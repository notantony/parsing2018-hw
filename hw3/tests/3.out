void max(int A, int B) {
    if ((A > B)) {
        std::cout << A;
    }
    else {
        std::cout << B;
    }
}
